### PR TITLE
update centos stream8

### PR DESCRIPTION
**Pull request by ebattat**
This PR contains:
1. Update vm to centos stream8 container disk
2. Update mssql kata timeout to 5400 instead of 3600 sec

### CODE REVIEW

File: benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py [hunk 1]
```diff
@@ -228,7 +228,7 @@ def __get_metadata(self, kind: str = None, database: str = None, status: str = N
         if run_artifacts_url:
             metadata.update({'run_artifacts_url': run_artifacts_url})
         if database:
-            metadata.update({'vm_os_version': 'centos8'})
+            metadata.update({'vm_os_version': 'centos-stream8'})
```
Comment (RobertKrawitz) [inline benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py:231]: This should really be fully qualified with the repo (I presume it's docker.io, which has its own issues in terms of throttling).  Likewise for everything else.  At a minimum, it needs to be cleaned up when you get a chance.

File: benchmark_runner/common/template_operations/templates/hammerdb/hammerdb_data_template.yaml [hunk 2]
```diff
@@ -38,21 +38,21 @@ template_data:
       requests_memory: 10Mi
   extra:
     mariadb:
-      db_image: centos8-mariadb103-container-disk:latest
+      db_image: centos-stream8-mariadb103-container-disk:latest
```
Comment (RobertKrawitz) [inline benchmark_runner/common/template_operations/templates/hammerdb/hammerdb_data_template.yaml:41]: All of these should use the quay.io/cloud-bulldozer image, not the default docker.io ones.

**Review by RobertKrawitz — COMMENTED** [review]
This might not be a bad opportunity once and for all to clean up the image names in the data templates.

File: benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py (hunk 1 above)
Comment (ebattat) [reply benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py:231]: Full path [here](https://github.com/redhat-performance/benchmark-runner/blob/b7858ee7f63aa56ddf13e582beabf5d4d8c2b6bf/benchmark_runner/common/template_operations/templates/hammerdb/internal_data/hammerdb_vm_template.yaml#L80)

File: benchmark_runner/common/template_operations/templates/hammerdb/hammerdb_data_template.yaml (hunk 2 above)
Comment (ebattat) [reply benchmark_runner/common/template_operations/templates/hammerdb/hammerdb_data_template.yaml:41]: Full path [here](https://github.com/redhat-performance/benchmark-runner/blob/b7858ee7f63aa56ddf13e582beabf5d4d8c2b6bf/benchmark_runner/common/template_operations/templates/hammerdb/internal_data/hammerdb_vm_template.yaml#L80)

Comment (RobertKrawitz) [reply benchmark_runner/common/template_operations/templates/hammerdb/hammerdb_data_template.yaml:41]: Yep, OK.

File: benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py (hunk 1 above)
Comment (RobertKrawitz) [reply benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py:231]: OK

**Review by RobertKrawitz — APPROVED** [review]
LGTM

**Pull request merged**